config/defaults.conf

```diff
@@ -223,11 +223,18 @@ tokenizer = "MosesTokenizer"  // The name of the tokenizer, passed to the Task c
                               // Use "" for edge probing tasks to use the original tokenization,
                               // or one of 'OpenAI.BPE', 'MosesTokenizer', 'bert-uncased-base',
                               // etc.
-word_embs = none  // The type of word embedding layer. Set to none when using ELMo, GPT, or BERT.
-                  // Options: none, scratch (i.e., trained from scratch), glove, fastText.
+
+input_module = "" // The word embedding layer, available values are the following:
+                  // gpt, glove, ELMo, scratch (i.e., trained from scratch), fastText. For BERT, 
+                  // specify the version of BERT. Available values for BERT are: bert-base-uncased, 
+                  // bert-large-cased. For more, see:
```

Comment:
You deleted an ellipsis, and now this could read as saying we only support these two BERT versions.